app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    lst = command.split(' ')
```

Comment:
```suggestion
    lst = command.split()
```
Space is the default argument for split